docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_retriggering_integration_tests.adoc

```diff
@@ -0,0 +1,35 @@
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+== Prerequisites
+- The snapshot for which tests will be retriggered has completed its initial tests
```

Comment:
```suggestion
- The snapshot must complete all its initial tests before you trigger subsequent tests.
```

Reply:
I tweaked this suggestion a tiny bit, let me know if its fine this way.